index.html

```diff
@@ -1,112 +1,157 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="UTF-8">
-  <title>Gridle</title>
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>

```

Comment:
It is best if you use a section as the first general tag in your application, rather than a div.